app/src/main/java/com/example/hungrywolfes/ui/searchScreen/SearchScreenFragment.kt

```diff
@@ -69,35 +66,31 @@ class SearchScreenFragment : Fragment() {
         val verticalDivider = DividerItemDecoration(requireContext(), RecyclerView.VERTICAL)
         val horizontalDivider = DividerItemDecoration(requireContext(), RecyclerView.HORIZONTAL)
 
-        getContextCompat(verticalDivider)
-        getContextCompat(horizontalDivider)
+        setContextCompat(verticalDivider)
+        setContextCompat(horizontalDivider)
 
         binding.recyclerViewSearched.adapter = searchedImagesAdapter
         binding.recyclerViewSearched.addItemDecoration(horizontalDivider)
         binding.recyclerViewSearched.addItemDecoration(verticalDivider)
     }
 
-    private fun getContextCompat(divider: DividerItemDecoration) {
+    private fun setContextCompat(divider: DividerItemDecoration) {
```

Comment:
Please read again the previous comment on this one and carefully check the bolded text.